pkg/kubelet/images/image_manager.go

```diff
@@ -104,12 +104,7 @@ func (m *imageManager) EnsureImageExists(ctx context.Context, pod *v1.Pod, conta
 	}
 
 	// If the image contains no tag or digest, a default tag should be applied.
-	image, err := applyDefaultImageTag(container.Image)
-	if err != nil {
-		msg := fmt.Sprintf("Failed to apply default image tag %q: %v", container.Image, err)
-		m.logIt(ref, v1.EventTypeWarning, events.FailedToInspectImage, logPrefix, msg, klog.Warning)
-		return "", msg, ErrInvalidImageName
-	}
+	image, _ := applyDefaultImageTag(container.Image)
```

Comment:
It seems to me that we should maybe consider applying default tag in API call.  

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/v1/defaults.go#L70